### PR TITLE
Add fake data for render

### PR DESCRIPTION
**Pull request by pftom**
1.add content.json under folder util
2.add fake `change` field for render every change file

### CODE REVIEW

File: renderer/src/App.js [hunk 1]
```diff
@@ -35,14 +35,14 @@ class App extends Component {
...
-      <div style={{ height: '100%' }}>
+      <div style={{ height: '100%', width: '100%', display: 'flex' }}>
         <Catalog 
-          catalogs={catalogs}
-          selectKey={this.state.selectKey}
-          updateSelect={this.updateSelect}
-        />
-        <Content 
-          content={nowRenderContent}
+            catalogs={catalogs}
```
Comment (mRcfps) [inline renderer/src/App.js]: Why change indentation here?

File: renderer/src/components/Content.js [hunk 2]
```diff
@@ -13,15 +13,20 @@ export default class Content extends Component {
...
+            <TabPane tab={diffItem.file} key={key}>
+              <ContentItem renderItem={diffItem.explain} />
+            </TabPane>
+          ))
+        }
+        </Tabs>
+      </div>
     );
   }
 }
```
Comment (mRcfps) [inline renderer/src/components/Content.js]: Later I'll add `.editorconfig` to enforce new line at the end of file.

Comment (pftom) [reply renderer/src/components/Content.js]: Oh, a good idea, I think

Comment (pftom) [reply renderer/src/components/Content.js]: I'll add it

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[*.js]
```
[the remark this answers is not in the document]
Comment (pftom) [reply .editorconfig]: Good, I will modify it. 👌

**Pull request closed without merging**